### ISSUE TITLE
network command parameters

### ISSUE THREAD

**Issue by laszewsk** [cloudmesh/client#100]
* [x] assess if we keep the following syntax: 

    network fixed-ip-get --cloud india 10.1.2.5

* [x] an alternative which may be more consitant with other commands and more readable is

  network get fixed ip --cloud=india 10.1.2.5
  network get fixed --cloud=india 10.1.2.5

  for the convenience of the user, we simply have the ip optional, so both of the above will work

* [x] provide reason why first should be used
* [x] provide reason why second should be used
* [x] make decision
* [x] implement

**Comment by gouravshenoy** [comment]
NetworkCommand now uses the following syntax(s):
```cmd
network get fixed ip --cloud india 10.1.2.5
OR
network get fixed --cloud india 10.1.2.5
```

**Comment by gouravshenoy** [comment]
This syntax is much more readable and provides user the flexibility to either provide *_ip_* or execute without providing *_ip_*, which is naturally implied.

Closing this issue now.
